components/brave_new_tab_ui/containers/newTab/settings.tsx

```diff
@@ -138,6 +142,14 @@ export default class Settings extends React.PureComponent<Props, {}> {
                 size='small'
               />
             </SettingsRow>
+            <SettingsRow>
```

Comment:
Since Together only shows if the user is in the US, should the setting _also_ be hidden if outside the US?

Reply:
I agree. @ryanml thoughts?

Reply:
yeah it should, good catch

Reply:
done